src/py21cmfast/src/UsefulFunctions.c

```diff
@@ -151,27 +151,30 @@ double HI_ion_crosssec(double nu);
 
 
 /* R in Mpc, M in Msun */
-double MtoR(double M){
+double MtoR(double M, struct FlagOptions *flag_options){
```

Comment:
I think it would be cleaner to have `MtoR` depend on the int variable `filter` rather than all of `flag_options`.